metadata.rb

```diff
@@ -6,4 +6,5 @@
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
 version          '0.2.0'
 
-depends 'ark'
+depends 'ark', "= 0.9.0"
+depends 'apt'
```

Comment:
Ah, I added it to .kitchen.yml, but forgot to remove it from metadata.rb (intermediary stage before I realised why that was a bad idea), apologies, fixing.